update-commit-statues-matching.py

```diff
@@ -0,0 +1,36 @@
+import github_utils
+import argparse
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--repository", "-r")
+    parser.add_argument("--commit", "-c")
+    parser.add_argument("--prefix", "-p")
+    args = parser.parse_args()
+
+    status_prefix = f"{args.prefix}/"
+
+    all_statuses = github_utils.get_combined_statuses(args.commit, args.repository).get(
+        "statuses", []
+    )
+    index = 0
+
+    for status in all_statuses:
+        if (
+            status["context"].startswith(status_prefix)
+            and status["context"].endswith("/rocm")
```

Comment:
@iarspider , instead of hardcoding `rocm` here pass it via command-line

Reply:
Should it be passed from [upstream job](https://cmssdt.cern.ch/jenkins/job/jenkins-elasticsearch-monitor/) or hardcoded in [a job that uses this script](https://cmssdt.cern.ch/jenkins/job/kill-stuck-pr-test/)?